### PR TITLE
Try `chai@next`

### PR DESCRIPTION
**Pull request by koddsson**
Trying out the `chai@5` prerelease so we can see if there are issues that we can feed back to the chai team.

### CODE REVIEW

**Comment by koddsson** [comment]
This is looking good! I'm thinking I'll let this sit until Chai@5 lands.

File: integration/test-runner/package.json [hunk 1]
```diff
@@ -14,7 +14,6 @@
   },
   "author": "modern-web",
   "homepage": "https://github.com/modernweb-dev/web/tree/master/packages/test-runner-integration-tests",
-  "main": "index.js",
```
Comment (thepassle) [inline integration/test-runner/package.json:17]: Why remove this?

File: package.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
     "type": "git",
     "url": "https://github.com/modernweb-dev/web.git"
   },
+  "type": "module",
```
Comment (thepassle) [inline package.json:9]: Why was this needed? I do think this can have impact on packages that use it

**Comment by thepassle** [comment]
This is cool dude, nice work. Changes look pretty straightforward (mostly import changes), nice.

File: integration/test-runner/package.json (hunk 1 above)
Comment (koddsson) [reply integration/test-runner/package.json:17]: It's unrelated to the chai changes. I was just getting warnings when running the tests:

```
(node:87634) [DEP0128] DeprecationWarning: Invalid 'main' field in '/src/web/integration/test-runner/package.json' of 'index.js'. Please either fix that or report it to the module author
```

**Comment by Westbrook** [comment]
while you're killing it here, any thoughts on a `mocha@10.x` PR? 😉

**Comment by koddsson** [comment]
> while you're killing it here, any thoughts on a `mocha@10.x` PR? 😉

Sure! I'll look into that 😄
